bijouxfantaisie/index.html

```diff
@@ -136,4 +136,10 @@
   }
 </script>
 <script src=/js/main.js></script>
+<script>
+  function preSubmitAmount(event) {
+    event.target.action = calculator.amount() <= 150
```

Comment:
It's actually great to have that specificity within the page and not in our main js.